tools/linter/codespell/.codespell.skip

```diff
@@ -12,3 +12,4 @@
 go.mod
 go.sum
 bin
+testdata
```

Comment:
this isn't right, we were linting `testdata` earlier

Reply:
let me ignore special file, configdump file will not pass codespell check.

Reply:
done

Reply:
I pulled in your branch and deleted the `tracer` that was causing the issue
```
diff --git a/internal/cmd/egctl/testdata/config/out.json b/internal/cmd/egctl/testdata/config/out.json
index d7bc3d3..1bd6f16 100644
--- a/internal/cmd/egctl/testdata/config/out.json
+++ b/internal/cmd/egctl/testdata/config/out.json
@@ -1316,10 +1316,6 @@
                                 "envoy.extensions.filters.network.thrift_proxy.router.v3.Router"
                             ]
                         },
-                        {
-                            "name": "envoy.dynamic.ot",
-                            "category": "envoy.tracers"
-                        },

```
this fixed `codespell`